disk-utils/mkfs.cramfs.8.adoc

```diff
@@ -62,14 +62,13 @@ This option is ignored. Originally the *-s* turned on directory entry sorting.
 *-z*::
 Make explicit holes.
 
-*-l*\fR[=\fImode\fR]::
+*-l* _mode_::
```

Comment:
The mode argument is optional, so [=mode] is expected there. I'll fix it before committing.